lib/modules/manager/cnb/readme.md

```diff
@@ -0,0 +1,21 @@
+The `cnb` manager updates Cloud Native Buildpacks project descriptor file (`project.toml`), which references builder and/or buildpack images by URIs.
+Updates are performed if the `project.toml` file is found and it conforms to the [project descriptor](https://github.com/buildpacks/spec/blob/main/extensions/project-descriptor.md) specification.
```

Comment:
```suggestion
Renovate can update a `project.toml` file if:

- It can find the file
- The file follows the [project descriptor](https://github.com/buildpacks/spec/blob/main/extensions/project-descriptor.md) specifications
```